packages/table/src/ExpandedContent/ExpandedContent.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo, useRef } from 'react';
+import { Transition } from 'react-transition-group';
+import { RowData } from '@tanstack/react-table';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { cellContentContainerStyles } from '../Cell/Cell.styles';
+import InternalRowBase from '../Row/InternalRowBase';
+
+import {
+  baseStyles,
+  expandedContentStyles,
+  expandedContentTransitionStyles,
+} from './ExpandedContent.styles';
+import { ExpandedContentProps } from './ExpandedContent.types';
+
+const ExpandedContent = <T extends RowData>({
+  row,
+  ...rest
+}: ExpandedContentProps<T>) => {
+  const contentRef = useRef<HTMLDivElement>(null);
+  const isExpanded = row.getIsExpanded();
+  const content =
+    row?.original?.renderExpandedContent &&
+    row?.original?.renderExpandedContent(row);
```

Comment:
`row?.original?.renderExpandedContent?.(row)`

Reply:
Without it, you still get the error `Cannot invoke an object which is possibly 'undefined'.ts(2722)`